### ISSUE TITLE
AgGrid.fsproj file reference package files from inside demo. Is this correct?

### ISSUE THREAD

**Issue by martinbryant** [CompositionalIT/feliz-ag-grid#15]
https://github.com/CompositionalIT/feliz-ag-grid/blob/main/src/Feliz.AgGrid.fsproj#L20-L25

**Comment by isaacabraham** [comment]
This doesn't feel right - I would prefer to duplicate them directly into the main package project.

**Comment by mattgallagher92** [comment]
I agree. At a glance I'm not even sure that they're necessary. Could probably just be removed from the file.

**Comment by Larocceau** [comment]
Only thing that needs to be done here still is remove the readme reference
